### DISCUSSION TITLE
Crawl working and search results recorded, but no results shown in search Docusaurus V3.0

### DISCUSSION THREAD

**Discussion by MatthewHawkins** [algolia/docsearch#2254] [Q&A]
Hi all,

I'm currently trying to integrate Algolia into my Docusaurus site. I was able to get my index name, search API key and app ID generated, but when I search on Docusaurus, I don't see any results. I'm able to see that the crawler indexes the site URLs, and I received an email about the searches I entered to test, but still see no results.

Has anyone else run into this? Any ideas what might be going on? Happy to provide additional information as needed - thanks in advance for any help!

**Comment by randombeeper** [comment]
Hello @MatthewHawkins, sorry I did not see this earlier. Chances are you are running into a bug that existed at that time. Maybe you've resolved it or moved on but if it is still outstanding issue for you, connect with me/us on Discord at https://alg.li/discord and we'll be happy to help you there! This discussion area is not well monitored.
